requirements.txt

```diff
@@ -0,0 +1 @@
+streamlit==1.16.0
```

Comment:
⚪ It would be better if you could use `>=` so users can install more recent version of streamlit

Reply:
It seems meaningful.
God Job...